amplipi/display/einkdisplay.py

```diff
@@ -150,17 +194,159 @@ def update_display(self, info: SysInfo = SysInfo(None, None, None), draw_base=Fa
 
   def pick_pass_font(self, password, max_length) -> ImageFont.FreeTypeFont:
     """Pick the password font so it fits in the display"""
+    max_size = self.pass_fontsize
+    min_size = math.floor(self.pass_min_fontsize)
     try:
-      for i in range(20, 10, -1):
+      for i in range(max_size, min_size, -1):
         font = ImageFont.truetype(self.fontname, i)
-        if font.getlength(password) <= max_length:
+        if font.getlength("Pass: " + password) <= max_length:
           return font
     except Exception as exc:
       raise Exception(f'Failed to load {self.fontname} font') from exc
     return ImageFont.truetype(self.fontname, 10)
 
 
-def get_info(iface, default_pass) -> SysInfo:
+def update_logs(status) -> None:
```

Comment:
What is the purpose of this? It looks like you're reinventing logging here. All the rest of our logs are captured by `journald` (it's just capturing `stdout` and `stderr`, nothing fancy) and are viewable with `journalctl -f` or by going to Settings->About->Logs in the AmpliPi web UI which pulls from the same journal service.

Also I tested it and the only log message I'm seeing in `display-logs.json` is "Starting Up". I tried changing the status and no other statuses show in that log file.

Reply:
This was one possible path I mentioned on day 2 as a persistent store of statuses to display. It may be vestigial now, particularly if we don't have external things reading or writing status, to this file or elsewhere - everything is simply represented as a state from the API.

Reply:
It would make a lot more sense to move the log file's information to the actual logs. I'm making that change now.